luigi/static/visualiser/js/visualiserApp.js

```diff
@@ -718,11 +718,21 @@ function visualiserApp(luigi) {
                         // Destination node may not be in tasks if this is an inverted graph
                         if (taskIdMap[task.inputQueue[i]] !== undefined) {
                             if (task.status === "DONE") {
-                                var durationTime = getFinishTime(tasks, task.inputQueue);
-                                g.setEdge(task.inputQueue[i], task.taskId, {
-                                    label: durationTime[task.inputQueue[i]] + " secs",
-                                    width: 40
-                                });
+                                var durations = getDurations(tasks, task.inputQueue);
+                                var duration = durations[task.inputQueue[i]]
+                                var oneDayInMilliseconds = 24 * 60 * 60 * 1000
+                                if (duration.getTime() < oneDayInMilliseconds) {
+                                    // Label task duration in stripped ISO format (hh:mm:ss:f)
+                                    g.setEdge(task.inputQueue[i], task.taskId, {
+                                        label: duration.toISOString().substr(11, 12),
+                                        width: 40
+                                    });
+                                } else {
+                                    g.setEdge(task.inputQueue[i], task.taskId, {
+                                        label: "> 24h",
```

Comment:
Can you remove this repetition?

Reply:
Sorry for the slow response. Fixed in the latest commit.